azure-automation-plus-dsc-lab/dsc/xActiveDirectory/Assert-ParentChildDomains.ps1

```diff
@@ -0,0 +1,92 @@
+[System.Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingComputerNameHardcoded', '')]
+[System.Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingConvertToSecureStringWithPlainText', '')]
+param()
+
+$secpasswd = ConvertTo-SecureString "Adrumble@6" -AsPlainText -Force
+$domainCred = New-Object System.Management.Automation.PSCredential ("sva-dscdom\Administrator", $secpasswd)
+$safemodeAdministratorCred = New-Object System.Management.Automation.PSCredential ("sva-dscdom\Administrator", $secpasswd)
+$localcred = New-Object System.Management.Automation.PSCredential ("Administrator", $secpasswd)
+$redmondCred = Get-Credential -Message "Enter Credentials for DNS Delegation: "
+$newpasswd = ConvertTo-SecureString "Adrumble@7" -AsPlainText -Force
```

Comment:
remove hardcoded password

Reply:
Did not author this script. Part of xActiveDirectoryModule.